src/test/java/org/openstreetmap/atlas/geography/MultiPolyLineTest.java

```diff
@@ -138,4 +138,17 @@ public void testCreateMultiPolyLineFromWKT()
         Assert.assertTrue(polyLines.contains(polyLine1));
         Assert.assertTrue(polyLines.contains(polyLine2));
     }
+
+    @Test
+    public void testEquals()
```

Comment:
Should we also test for a case where two ```MultiPolyLine``` are different?

Reply:
Sure, I will add a case for that.